src/selectors/index.js

```diff
@@ -48,6 +48,14 @@ const defaultRequest = Immutable({
 // Store dataKey-specific selectors created on demand
 const selectByKey = { data: {}, obj: {}, owr: {}, ref: {}, req: {} }
 
+export const selectCachePurge = () => {
```

Comment:
Super nit: since we have the `purgeDenormalizationCache`, can we name this similarly (e.g. `purgeSelectorCache` or whatever).